src/site/content/en/blog/test-post/index.md

```diff
@@ -513,3 +601,74 @@ porta dolor erat, vel molestie dolor posuere in. Nam vel elementum augue. Nam
 quis enim blandit, posuere justo dignissim, scelerisque diam. Fusce aliquet urna
 ac blandit ullamcorper. Proin et semper nibh, sit amet imperdiet velit. Morbi at
 quam sem.
+
+{% Details %}
```

Comment:
The margin and border between these details looks off 
![image](https://user-images.githubusercontent.com/11811422/151467254-6d1dc98b-c90a-4673-80d2-bd9846cea4e7.png)

Reply:
Yes, this is why these elements were added to test post, to catch all regressions we have elswhere in the code.